sso/backend/src/main/java/com/sso/security/config/JwtFilterConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.sso.security.config;
+
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class JwtFilterConfig {
+    @Bean
+    public FilterRegistrationBean<JwtFilter> jwtFilter() {
+        System.out.println("jwtt token");
+        FilterRegistrationBean filter= new FilterRegistrationBean();
+
+        filter.setFilter(new JwtFilter());
+        filter.addUrlPatterns("/users/*");
+        filter.addInitParameter("exclusions", "/users/save/*,/users/login/*");
```

Comment:
if you  are excluding the end points here, you can probably remove them from isExcluded method in JWTFilter class